Consumer-LOS/Configurations/standard_configuration.json

```diff
@@ -0,0 +1,271 @@
+        "label": "CL Purpose"
+    },
+
+    {
+        "Sequence":5,
+        "Object API Name":"clcommon__Document_Definition__c",
+        "External Id Field":"genesis__External_ID__c",
+        "label": "Document Definition" 
+    },
+
+    {
+        "Sequence":6,
+        "Object API Name":"clcommon__Rule_Set__c",
+        "External Id Field":"clcommon__External_ID__c",
+        "label": "Rule Set"
+    },
+    {
+        "Sequence": 7,
+        "Object API Name": "clcommon__Approval_Configuration__c",
+        "External Id Field": "solcommon__External_Id__c",
+        "label" : "Approval Configuration"
+    },
+
+    {
+        "Sequence":8,
+        "Object API Name":"clcommon__Reason_Code__c",
+        "External Id Field":"clcommon__External_ID__c",
+        "label": "Reason Code"
+    },
+
+    {
+        "Sequence":9,
+        "Object API Name":"clcommon__Reciprocal_Role__c",
+        "External Id Field":"clcommon__External_ID__c",
+        "label": "Reciprocal Role"
+    },
+
+
+    {
+        "Sequence":15,
+        "Object API Name":"clcommon__Party_Type__c",
+        "External Id Field":"clcommon__External_ID__c",
+        "label": "Party Type"
+    },
+
+    {
+        "Sequence":16,
+        "Object API Name":"clcommon__CL_Product__c",
+        "External Id Field":"genesis__External_ID__c",
+        "label": "CL Product"
+    },
+
+    {
+        "Sequence":17,
+        "Object API Name":"clcommon__Rule__c",
+        "External Id Field":"clcommon__External_ID__c",
+        "label": "Rule"
+    },
+
+
+    {
+        "Sequence":30,
+        "Object API Name":"genesis__Credit_Rating_Matrix__c",
+        "External Id Field":"genesis__External_ID__c",
+        "label": "Credit Rating Matrix"
+    },
+
+    {
+        "Sequence":31,
+        "Object API Name":"genesis__Rate_Card_Setup_Header__c",
+        "External Id Field":"genesis__External_ID__c",
+        "label": "Rate Card Setup"
+    },
+
+    {
+        "Sequence":32,
+        "Object API Name":"genesis__Rate_Card_Setup_Detail__c",
+        "External Id Field":"genesis__External_ID__c",
+        "label": "Rate Card Setup Detail"
+    },
+
+
+    {
+        "Sequence":40,
+        "Object API Name":"genesis__Department__c",
+        "External Id Field":"genesis__External_ID__c",
+        "label": "Department"
+    },
+
+    {
+        "Sequence":41,
+        "Object API Name":"genesis__Task_Setup__c",
+        "External Id Field":"genesis__External_ID__c",
+        "label": "Task Setup"
+    },
+
+    {
+        "Sequence":42,
+        "Object API Name":"genesis__Product_Department_Junction__c",
+        "External Id Field":"genesis__External_ID__c",
+        "label": "Product Department Association"
+    },
+
+    {
+        "Sequence":43,
+        "Object API Name":"genesis__ProductDepartment_Task_Junction__c",
+        "External Id Field":"genesis__External_ID__c",
+        "label": "Product Department Task Association"
```

Comment:
Remove the spacing in the label to match it with the object label as defined by genesis : "ProductDepartment Task Association"